src/init.lua

```diff
@@ -0,0 +1,190 @@
+--!strict
+--[[
+
+        NametagPlus.addTag(player): BillboardGui
+        NametagPlus.getTag(player): BillboardGui
+        NametagPlus.changeTag(tag, ...): nil
+        NametagPlus.linkTag(player, tag, groupId?): nil
+
+--]]
+
+local config = require(script.Configuration)
+local utils = require(script.Utilities)
+
+local holder = utils.constructFromProperties("Folder", config.defaults.holder)
+
+local nametagPlus = {}
+
+-->> Functions
+local function addTag(player: Player): BillboardGui
+	-- Create the essentials for the nametag
+	-- Overriding the type is essential for typechecking.
+	local billboard = utils.constructFromProperties("BillboardGui", config.defaults.billboard) :: BillboardGui
+	local frame = utils.constructFromProperties("Frame", config.defaults.frame) :: Frame
+	local name = utils.constructFromProperties("TextLabel", config.defaults.name) :: TextLabel
+	local mainStat = utils.constructFromProperties("TextLabel", config.defaults.mainStat) :: TextLabel
+	local extraStat = utils.constructFromProperties("TextLabel", config.defaults.extraStat) :: TextLabel
+	local healthBar = utils.constructFromProperties("Frame", config.defaults.healthBar) :: Frame
+	local healthBackground = healthBar:Clone()
+
+	-- Initial setup of components
+	billboard.Enabled = true
+	healthBackground.BackgroundTransparency = 0.5
+	healthBackground.Name = "HealthBackground"
+	healthBackground.ZIndex = 1
+
+	if player ~= nil then
+		local finalName = utils.waitForProperty(player, "Name")
```

Comment:
It looks like this function was removed from Utilities. Could you either add it back to Utilities or find a work around here? Whenever I try to call addTag in Studio this line errors.